src/components/shipModal.html

```diff
@@ -0,0 +1,27 @@
+<div class="modal noselect fade" id="ShipModal" tabindex="-1" role="dialog" aria-labelledby="ShipModalLabel">
```

Comment:
What is this file?

Reply:
Modal that opens when you click one of the docks at Olivine or Vermillion. Accessed through the dungeon square next to those cities.

Reply:
Oh huh I put it at the wrong location. I meant to ask about the test.html file

Reply:
oh, whoops, that was just so I could see the diff between the map in develop and my KantoSVG. It shouldn't have been in the pr